metadata-service/war/src/main/resources/boot/groups.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "urn": "urn:li:corpgroup:organization-admins",
+    "info": {
+      "displayName": "Organization Admins",
+      "description": "Top level DataHub administrators. They have ALL privileges.",
+      "admins": [],
+      "members": [],
+      "groups": []
+    }
+  },
+  {
+    "urn": "urn:li:corpgroup:metadata-stewards",
+    "info": {
+      "displayName": "Metadata Stewards",
+      "description": "Overseers of all metadata assets across DataHub.",
```

Comment:
What can these guys do? Consider writing that here!

Reply:
Made that clearer